docs/fundamentals/introduction.md

```diff
@@ -10,34 +10,34 @@ Veramo is in a public beta. There will be some breaking changes in the coming mo
 
 :::
 
-Veramo is a JavaScript framework that makes it easy for anyone to use cryptographically verifiable data in their applications. It was designed to make it easy for developers who want to use DIDs, verifiable credentials, and other data-centric protocols to bring next-generation features to their users.
+Veramo is a JavaScript framework that makes it easy for anyone to use cryptographically verifiable data in their applications. It was designed to make it easy for developers who want to use DIDs, verifiable credentials, and data-centric protocols to bring next-generation features to their users.
 
 These docs are written for anyone wishing to learn more about the grounding principles behind verifiable data and develop a deeper understanding of the Veramo framework to build next-generation apps that leverage the many benefits of trust networks.
 
 ## How to use these docs
 
 If you are new to the concept of verifiable data, it would be best to read the fundamentals section linearly as each section builds on knowledge and terminology gained in the previous.
 
-If you are more experienced with verifiable data concepts and or development in general, you can dive into the architecture section where more specifics of how the internals work is covered, or you could [deploy your own cloud agent](/docs/guides/cloud_agent). You can also jump over to the [API reference](/docs/api/daf-core) section. Links and more info about these packages can also be found in the plugins section.
+If you are more experienced with verifiable data concepts and/or development in general, you can dive into the architecture section where more specifics of how the internals work is covered, or you could [deploy your own cloud agent](/docs/guides/cloud_agent). You can also jump over to the [API reference](/docs/api/daf-core) section. Links and more info about these packages can also be found in the [Plugins](/docs/api/agent/plugins) section.
```

Comment:
This is a broken link ~ Use `/docs/agent/plugins_list` as we can link to the api section from each package there

Reply:
done